README.md

```diff
@@ -1,8 +1,34 @@
 # Meteor on Google Compute Engine
 Install scripts to run Meteor on Google Compute Engine.
 
+## Option 1: using MUP
+This is a fast solution, but doesn't work on Windows yet :(. It uses Meteor UP (https://github.com/arunoda/meteor-up), has SSH Key authentication for GCE, and only works on Ubuntu images.
 
-## Usage
+1. Create a GCE VM instance via:
+  - http://cloud.google.com/console:
+    + create a new project or choose an existing one
+    + in `Compute` - `Compute Engine` - `VM instances` click `New instance`
+    + check `Allow HTTP traffic` and `Allow HTTPS traffic`
+    + MUP only works on Ubuntu. Choose an ubuntu image, preferably the latest LTS (long-term-support) version, currently 1404
+    + use a Static IP address so that you can point your DNS to it
+  - commandline: `gcloud compute --project "my-meteor-project" instances create "meteor-vm" --zone "europe-west1-d" --machine-type "n1-standard-1" --network "default" --address 104.155.35.68 --maintenance-policy "MIGRATE" --scopes "https://www.googleapis.com/auth/devstorage.read_only" "https://www.googleapis.com/auth/logging.write" --tags "http-server" "https-server" --image "https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/ubuntu-1404-trusty-v20150316" --boot-disk-type "pd-standard" --boot-disk-device-name "meteor-vm"`
```

Comment:
I think you're missing a persistent disk for Mongo now. That might not actually be a problem, but is it really what you want?